Airports/EGGW/Stars.txt

```diff
@@ -1,44 +1,59 @@
-STAR:EGGW:25:ABBOT1Z:ABBOT TABIS BUSTA LOREL
-STAR:EGGW:07:ABBOT1Z:ABBOT TABIS BUSTA LOREL
-STAR:EGGW:25:AVANT1L:AVANT OCK VATON BPK BKY BUSTA LOREL
-STAR:EGGW:07:AVANT1L:AVANT OCK VATON BPK BKY BUSTA LOREL
-STAR:EGGW:25:BANVA1L:BANVA UNDUG MAY VATON BPK BKY BUSTA LOREL
-STAR:EGGW:07:BANVA1L:BANVA UNDUG MAY VATON BPK BKY BUSTA LOREL
-STAR:EGGW:25:BARMI1A:BARMI MEGEL DITOB LAPRA ABBOT
-STAR:EGGW:07:BARMI1A:BARMI MEGEL DITOB LAPRA ABBOT
-STAR:EGGW:25:BEDEK1L:BEDEK NIGIT OCK VATON BPK BKY BUSTA LOREL
-STAR:EGGW:07:BEDEK1L:BEDEK NIGIT OCK VATON BPK BKY BUSTA LOREL
-STAR:EGGW:25:DET2A:DET LOFFO ABBOT
-STAR:EGGW:07:DET2A:DET LOFFO ABBOT
-STAR:EGGW:25:FINMA1L:FINMA BOMBO BKY BUSTA LOREL
-STAR:EGGW:07:FINMA1L:FINMA BOMBO BKY BUSTA LOREL
-STAR:EGGW:25:IDESI1A:IDESI LAPRA ABBOT
-STAR:EGGW:07:IDESI1A:IDESI LAPRA ABBOT
-STAR:EGGW:25:LOGAN2A:LOGAN CLN ABBOT
-STAR:EGGW:07:LOGAN2A:LOGAN CLN ABBOT
-STAR:EGGW:25:LOREL5F:LISTO PEDIG ROGBI CLIPY BOMBO BKY BUSTA LOREL
-STAR:EGGW:07:LOREL5F:LISTO PEDIG ROGBI CLIPY BOMBO BKY BUSTA LOREL
-STAR:EGGW:25:LOREL3G:MCT PEDIG ROGBI CLIPY BOMBO BKY BUSTA LOREL
-STAR:EGGW:07:LOREL3G:MCT PEDIG ROGBI CLIPY BOMBO BKY BUSTA LOREL
-STAR:EGGW:25:LOREL2H:HON CLIPY BOMBO BKY BUSTA LOREL
-STAR:EGGW:07:LOREL2H:HON CLIPY BOMBO BKY BUSTA LOREL
-STAR:EGGW:25:SILVA1L:SILVA BOMBO BKY BUSTA
-STAR:EGGW:07:SILVA1L:SILVA BOMBO BKY BUSTA
-STAR:EGGW:25:TELTU1L:TELTU VATON BPK BKY BUSTA LOREL
-STAR:EGGW:07:TELTU1L:TELTU VATON BPK BKY BUSTA LOREL
-;Withdrawn 2019/05
-STAR:EGGW:25:#LOREL2D:GIBSO BEGTO AVANT OCK VATON BPK BKY BUSTA LOREL
-STAR:EGGW:07:#LOREL2D:GIBSO BEGTO AVANT OCK VATON BPK BKY BUSTA LOREL
+STAR:EGGW:22:ABBOT1Z:ABBOT TABIS BUSTA LOREL
+STAR:EGGW:04:ABBOT1Z:ABBOT TABIS BUSTA LOREL
+STAR:EGGW:22:AVANT1L:AVANT OCK VATON BPK BKY BUSTA LOREL
+STAR:EGGW:04:AVANT1L:AVANT OCK VATON BPK BKY BUSTA LOREL
+STAR:EGGW:22:BANVA1L:BANVA UNDUG MAY VATON BPK BKY BUSTA LOREL
+STAR:EGGW:04:BANVA1L:BANVA UNDUG MAY VATON BPK BKY BUSTA LOREL
+STAR:EGGW:22:BARMI2A:BARMI MEGEL DITOB LAPRA ABBOT
+STAR:EGGW:04:BARMI2A:BARMI MEGEL DITOB LAPRA ABBOT
+STAR:EGGW:22:BEDEK1L:BEDEK NIGIT OCK VATON BPK BKY BUSTA LOREL
+STAR:EGGW:04:BEDEK1L:BEDEK NIGIT OCK VATON BPK BKY BUSTA LOREL
+STAR:EGGW:22:BKY1X:BKY ADNAM ABBOT
+STAR:EGGW:04:BKY1X:BKY ADNAM ABBOT
+STAR:EGGW:22:BPK1X:BPK ADNAM ABBOT
+STAR:EGGW:04:BPK1X:BPK ADNAM ABBOT
```

Comment:
BPK1X doesn't exist as far as I can tell

Reply:
it does seem luton doesn't have any stack swap stars